guides/common/modules/ref_dns-options-for-network-configuration.adoc

```diff
@@ -2,28 +2,36 @@
 = DNS Options for Network Configuration
```

Comment:
Please read through this carefully to see if it's clear. I still think that long term we should move to documenting all dns_* providers like we now did with `dns_libvirt` and `dns_powerdns`. When an option is relevant, document it in the section. However, for now I think this improves it.